account-kit/infra/src/client/smartAccountClient.ts

```diff
@@ -174,21 +171,24 @@ export function createAlchemySmartAccountClient(
       ? alchemyUserOperationSimulator(config.transport)
       : undefined,
     signUserOperation: config.signUserOperation,
-  }).extend(alchemyActions);
-
-  if (config.account && isSmartAccountWithSigner(config.account)) {
-    config.transport.updateHeaders(getSignerTypeHeader(config.account));
-  }
-
-  return {
-    ...scaClient,
-    [ADD_BREADCRUMB](breadcrumb: string) {
+    // @ts-expect-error
+    addBreadcrumb(crumb: string) {
       const newTransport = alchemy({ ...config.transport.config });
```

Comment:
config.transport is always AlchemyTransport here. AlchemyTransport ships with the `transport` function itself and the `config` that was passed to `alchemy(...)` which is why we can recreate it using the original config here. The rest is handled down stream for us